internal/pkg/levee/testdata/src/nopanic.com/core/core.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+type Source struct {
+	Data string
+	ID   int
+}
+
+func Calls() {
```

Comment:
This name should indicate what the purpose of this function is, e.g. `TestPanickingOnTaintedValueIsAllowed`.

Reply:
Updated to TestPanickingOnTaintedValueIsAllowed.